frontend/tsconfig.json

```diff
@@ -3,24 +3,31 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
what does this do?

Reply:
i think i turned it off when i was having some type issues, i should probably turn it back on....